lib/cocoapods/command/setup.rb

```diff
@@ -44,6 +46,26 @@ def run
         UI.puts 'Setup completed'.green
       end
 
+      def check_xcode_license_approved
+        unless xcode_license_approved?
+          UI.puts xcode_license_message.red
```

Comment:
- ¿do you propose adding that line, or replacing the `UI.puts` call?
- ¿what's the ramifications of that line? ¿who catches it?